RELEASE.md

```diff
@@ -80,61 +106,75 @@ Please make sure you have these tools installed before starting this process:
 
-    -   Sync with solution engineers to do acceptance testing, a clean installation on this phase is recommended.
+-   [ ] 7.  If necessary address different stakeholders and ask for help testing the new release.
 
-    -   If you find any critical path that might have been affected, make sure you add/edit the tests in our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/).
+-   [ ] 8.  Sync with solution engineers to do acceptance testing, a clean installation on this phase is recommended.
 
-    -   Inform the release manager "#enterprise-releases" on the progress/findings of the testing on the release.
+-   [ ] 9.  If you find any critical path that might have been affected, make sure you add/edit the tests in our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/).
 
+-   [ ] 10. Inform the Release Manager on #enterprise-releases about the progress/findings of the testing on the release.
 
--   [ ] 10.  Approval by QA
+-   [ ] 11. Approval by QA
 
     Involve the QA stakeholders in the release process.
 
     Remind them that this release candidate is available for testing in the [release environment](./README.md#Development).
 
     Should all of these stakeholders be happy, a go-ahead of the release should be given by clicking the Manual Approval step for QA in the CircleCI workflow of your release branch.
 
--   [ ] 11.  Approval by Solutions Engineers
+-   [ ] 12. Approval by Solutions Engineers
 
     Involve the Solutions Engineers stakeholders in the release process.
 
-    Remind them that this release candidate is available for testing in the [release environment](./README.md#Development).
+    Inform them that this release candidate is available for testing in the [release environment](./README.md#Development). At this point the Solutions Architects should:
+
+    -   [ ] 12.1.  Perform a fresh installation of the release candidate
+
+    -   [ ] 12.2.  Perform an upgrade on an existing installation
 
-    At this point it may be relevant to test a fresh installation of the release candidate as well as an upgraded installation.
+    -   [ ] 12.3.  Should all of these stakeholders be happy, a go-ahead of the release should be given by clicking the Manual Approval step for the Solutions Engineers in the CircleCI workflow of your release branch.
 
-    Should all of these stakeholders be happy, a go-ahead of the release should be given by clicking the Manual Approval step for the Solutions Engineers in the CircleCI workflow of your release branch.
-    
     After this Manual Approval on CircleCI the workflow will promote the RC to the [stable](https://charts.codacy.com/stable/api/charts) channel.
 
-    If all is good give a public OK to the release.
-    The final version will be `6.0.0`.
+## 4. Launching the new release
```

Comment:
In the near future I hope to include a step for publishing the release notes in this stage of the release process.